MapboxNavigation/PollyVoiceController.swift

```diff
@@ -46,7 +47,7 @@ public class PollyVoiceController: RouteVoiceController {
     
     var spokenInstructionsForRoute = NSCache<NSString, NSData>()
     
-    var previousInstruction: String?
+    var previousInstruction: SpokenInstruction?
```

Comment:
Rename this property to `lastSpokenInstruction`. Between spoken instructions, it makes more sense to talk about the last instruction that was said than the “previous” one.